cpp/CMakeLists.txt

```diff
@@ -171,17 +175,15 @@ target_compile_definitions(
 
 # Enable RMM if necessary
 if(UCXX_ENABLE_RMM)
-    target_compile_definitions(ucxx PUBLIC UCXX_ENABLE_RMM)
+    target_link_libraries(ucxx PUBLIC rmm::rmm)
+    target_compile_definitions(ucxx PUBLIC UCXX_ENABLE_RMM "SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL}")
 endif()
 
 # Define spdlog level
```

Comment:
Whoops this was my fault. Please move this comment to line 179.

Reply:
Done in https://github.com/rapidsai/ucxx/pull/167/commits/1b3977f3f7b73a75271c3c540113d142548bc985